dev-util/gammaray/gammaray-2.9.0.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+QT_MINIMAL="5.9"
```

Comment:
I don't see the actual minimum dependency in CMakeLists.txt, where did you?

Reply:
I will check this and the other PR in a few hours.

Reply:
Actually neither [here](https://github.com/KDAB/GammaRay/wiki/Roadmap), nor [here](https://www.kdab.com/gammaray-2-9-0-release/) there is a strict requierement for Qt-5.9. It just mentions that Qt5.9 enables some new features.

I will remove it.

Reply:
It seems this is about runtime capabilities that become available whenever it is started with QtQuick-5.9 available.